base-containers/base/bin/ssh_student

```diff
@@ -0,0 +1,35 @@
+#! /bin/python3
+# -*- coding: utf-8 -*-
+#
+# This file is part of INGInious. See the LICENSE and the COPYRIGHTS files for
+# more information about the licensing of this file.
+import argparse
+import sys
+
+from inginious_container_api import run_student
```

Comment:
As you have defined `ssh_student`, maybe it would be more coherent to import it here instead of `run_student`.